### PR TITLE
Layout for Risk History (EXPOSUREAPP-4577)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Adds layouts needed to display the risk history within the contact diary

### CODE REVIEW

**Comment by BMItter** [comment]
looking good so far for first version.
could you provide a string res too(or did the process changed)? maybe first version according to ticket or our designtool

**Comment by axelherbstreith** [comment]
> looking good so far for first version.
> could you provide a string res too(or did the process changed)? maybe first version according to ticket or our designtool

Yes, currently working on that 👍

**Review by BMItr — DISMISSED** [review]
all right for first version. may be pollished later.
Thx

File: Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item_risk.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        android:id="@+id/contact_diary_overview_element_nested_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:focusable="true"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item_risk.xml:32]: No need to nest a constraint layout within a constraint layout - especially if the inner one is the only child, you can drop the outer one.

**Review by BMItr — APPROVED** [review]

File: Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item_risk.xml (hunk 1 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item_risk.xml]: Good point! Is removed 👍

**Review by ralfgehrer — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA approved

**Pull request merged**